libs/core/src/lib/select/option/option.component.ts

```diff
@@ -47,7 +90,7 @@ let optionUniqueId: number = 0;
     `]
 
 })
-export class OptionComponent implements OnInit, OnDestroy {
+export class OptionComponent implements FocusableOption, OnDestroy, AfterViewChecked {
```

Comment:
Let's follow [contribution guidelines](https://github.com/SAP/fundamental-ngx/wiki/Code-Contribution-Guideline) on order of declarations

Reply:
@JKMarkowski I must have missed this WIKI page, we should probably put this into PR template, we refer only to MD files. But you are right I see in the COntributing.md

Reply:
Addressed